style.css

```diff
@@ -0,0 +1,94 @@
+.site-header {
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 36px 10px 27px;
+}
+
+.site-header a {
+  font-family: 'Roboto', serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  text-decoration: none;
+}
+
+.site-header .nav-menu img {
+  height: 26px;
+  width: 26px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #1c1a19;
+  background-image: url(./images/bgimg.png);
+  background-repeat: no-repeat;
+  background-position: 100% -70%;
+}
```

Comment:
- [ ] [OPTIONAL] It would be better if you placed these styles at the beginning because they are the first styles for the entire page 😃.